guides/assets/stylesrc/_main.scss

```diff
@@ -17,7 +17,15 @@ body.guide {
 
   -webkit-tap-highlight-color:rgba(38, 27, 35, 0);
 
-  // ----------------------------------------------------------------------------
+  html[dir="rtl"] & {
+    direction: rtl;
+
+    p {
+      text-align: right;
+    }
+  }
```

Comment:
Since this is similar/related to the font declarations at the bottom, and those also use `body.guide`, let's try and group them together.

How about something like this at the bottom:

```scss
html[dir="rtl"] {
  body.guide {
    // these declarations
  }

  &:lang("ar"), &:lang("fa") {
    body.guide { 
      // ... font declarations 
    }
  }

  &:lang("foo")...
}
```

So we have less spread changes related to dir=rtl.

Reply:
Cool, yeah I was mainly looking for this block really which seemed super related to the one at the bottom when I reviewed. :+1: